src/components/Playerbar.vue

```diff
@@ -1,12 +1,9 @@
 <template>
-  <div class="footer flex relative z-20 h-20 border-t border-default">
-    <div class="flex flex-none items-center w-80">
-      <span class="icon text-3xl ml-12 li-previous text-prevnext" @click="prevTrack()" />
-      <span
-        class="icon text-3xl ml-6 mr-6 li-play play text-play hover:text-play-hover"
-        :class="isPlaying ? 'li-pause' : 'li-play'"
-        @click="playPauseToggle()" />
-      <span class="icon text-3xl mr-4 li-next text-prevnext" @click="nextTrack()" />
+  <div class="footer flex flex-wrap gap-y-4 relative z-20 border-t border-default">
```

Comment:
New playbar UI layout seems too high and play control buttons are too big.
Please leave playbar UI unchanged, because redesign needs to adjust whole component design.